### ISSUE TITLE
[BUG]: Final Fantasy X Tidus Falls Through Lift

### ISSUE THREAD

**Issue by vcx33** [PCSX2/pcsx2#8228]
### Describe the Bug

When using the lift in the omega ruins, tidus falls through during his jump animation

Bug:
https://user-images.githubusercontent.com/118646244/220760903-795ecea1-f77a-4269-9f35-7e42954485bd.mp4

Correct behavior:
https://youtu.be/x5cI9PNjMUQ?t=919

Changing FPU values (rounding/clamping) does nothing

### Reproduction Steps

go to the omega ruins and step on the lift

### PCSX2 Revision

1.7.4126

### Operating System

Linux (64bit) - Specify distro below

**Comment by BParks21** [comment]
I'm having an issue in FF X-2 when flying enemies are killed they fall upward to their death instead of falling to the ground. Wonder if it's related? I'll probably open an issue for that as well.

**Comment by Mrlinkwii** [comment]
Can this be rechecked on latest release

**Comment by bigol83** [comment]
Still the same issue. Tried with every single advanced setting in EE and VU, it always has this issue.

**Comment by refractionpcsx2** [comment]
Can we get a modern savestate from before she falls through the floor? No guarantees of fixing it, but would like to see it

**Comment by bigol83** [comment]
This one should work for FF X PAL UK 

[final-fantasy-x.10053.zip](https://github.com/PCSX2/pcsx2/files/13938205/final-fantasy-x.10053.zip)

Use the teleport sphere to Omega Ruins, put a no Encounter armor, and reach the elevator. That's how i tested it.

**Comment by refractionpcsx2** [comment]
Teleport sphere? You'll have to instruct me like I've never played it :P

**Comment by bigol83** [comment]
I didn't know you already played it. 😜

**Comment by refractionpcsx2** [comment]
[ffx.zip](https://github.com/PCSX2/pcsx2/files/13940239/ffx.zip)

here's that save on a memcard, you go to the airship, talk to cid on the left, then choose to go to the Omega Ruins, go immediately down the left/west path, follow it round, then I think you go right at the fork, and there's a little floating platform thing you stand on as you go around, that's it.

I messed with game fixes, tried poking some of the code, didn't get anywhere though :(

**Comment by refractionpcsx2** [comment]
okay so this is fixed by setting EE Division Rounding to Negative makes the animations work in both directions! (Take that, Play! and Dobie lol).  Be aware that you will need to go via memcard, a savestate will have already stored a bad state for it.

However I don't know what affect it has on the rest of the game, it may make things explode, it'll need testing throughout.

**Comment by vcx33** [comment]
This fixes the lift bug, but not when EE rounding mode is set to positive which is required to fix another FPU bug (cid running in place during the airship cutscene liftoff). 
After testing a bit i found an FPU config that fixes pretty much every known FPU related bug, like characters and enemies facing the wrong way in battle, characters turning invisible during cutscenes, characters running into walls.

EE Rounding Mode: Negative
EE Division Rounding Mode: Chop/Zero
EE Clamping Mode: Full
 
VU rounding options are set to the default ones

**Comment by refractionpcsx2** [comment]
I believe EE division chop zero doesn't work going back down the lift

**Comment by vcx33** [comment]
settting EE division to chop/zero fixed the lift bug on my end

**Comment by refractionpcsx2** [comment]
> settting EE division to chop/zero fixed the lift bug on my end along with the other EE FPU settings i posted

Going up AND back down on the lift?

**Comment by vcx33** [comment]
yes up and down, you also dont need to reset the level you can just move forward a bit step on the warp pad and then come back

**Comment by refractionpcsx2** [comment]
alright, I just tried it, can confirm it seems be okay

**Comment by vcx33** [comment]
this EE FPU config still needs thorough testing to see if it breaks something else.

**Comment by vcx33** [comment]
Setting EE Division Rounding Mode to negative or chop/zero breaks a number of cutscenes, like the one where tidus washes up on besaid beach.
I guess FFX will always have FPU glitches one way or another.

**Comment by bigol83** [comment]
Just tested this: with default settings, meaning with EE recompiler enabled this is the behaviour

https://github.com/user-attachments/assets/240f99c7-5ec6-4e26-9bc9-3d741a89b93c

With EE recompiler disabled it has no issues 

https://github.com/user-attachments/assets/7af5b4db-b8b6-46cd-b17f-aa5e637b456f
